docs/en/relying-party-solution.rst

```diff
@@ -3,11 +3,50 @@
 
-.. include:: relying-party-entity-configuration.rst
+Logical architecture is composed of the following elements:
+
+- **Trust Anchor**: It represents the root of trust for the entire system and consents to Relying Parties to federate in the IT Wallet ecosystem. Trust Anchor has PKI capabilities and issues Registration Certificates during the Relying Party onboarding process. 
+- **Relying Party**: Organizational Entity that provides Relying Party Solutions with the aim of authenticating Users and/or verifying Digital Credentials presented by Users through their Wallet Instances. A Relying Party may provide:
+   
+   - **Relying Party Backend**: It handles Relying Party Instances registration and the relative certificate management. It MUST obtain a Registration Certificate by a TA or its intermediary, and it MUST have PKI capabilities to issue Access Certificates to its Relying Party Instances. It also may implement additional web services and business logic for its own purposes and use cases.
+   - **Relying Party Instance**: It is a frontend provided to Users in order to access Relying Party Services, and it may be provided as:
+      
+      - **Web Client**.
+      - **Mobile Application**.
+
+The following diagram depicts the Relying Party Solution High Level Architecture.
+
+
+.. figure:: ../../images/relying_party_solution_high_level_architecture.svg
+   :width: 100% 
+   :alt: The image illustrates the Relying Party Solution and its relations and interactions within the ecosystem.
+   :target: https://www.plantuml.com/plantuml/svg/jLHDRziy3BxxL_3DfIdmfhdij5FJD42DhPZHfYXsCcIPIOJ9aY39pOAX_tsadDZDl6BOOGOIcHG_deSVlWvH-DWsEljF6QdR6c4NemiVvtClzbTR5NTjrGRqqfg89bv9syoT5ePzPY7MMbNpvOTPmQgd-y_pHeI8dbJbqZRE6lPn73-xoszNvUDwzR3wilvQhAQNMNO1jxXH1a78Q7q0OMe81mhGXAn07woPSkx_OV94nuJE5Lcm2lQ43FBrx5beZN52mJAWfqzQhhx7QVHjYAKmScSvo9f5MB2OWl4l3w7500-uLI5w4PKri4GVrKP94R73vBnzKJK9nQSSf72YKbf5HOgDzH1t29HHUHm00KhRCKdGiXlRcbhumzIZFYYpBPtmyL1MHpK1UUWkeE0BUBIwPIoJI_XnJcpgNLa9-GrDFVAwwVGdRdU4NJEv0d8JeZ0ImRybojpaG1OQOSGM6-uPv8tJwJK03O0o609ekrKoSGuE9NjnCq2AW85u0ZtfRs3hIdNC_2xN7blFTSHtZrKfjIA9d9-nWcKIZZnPCwi8GmqRdyQq1aM7tmY92IrF3uWsxhXDSk1-gxOg2WrBZpLVqUiB-bIDhlY_XzDOwZ-MmZbAVfy4M3YdJsuCgx_vbyXUizSiUqnZbkPzdsQlBpcN9cc64XzNwIBq6JgBXFdjHdhKALFOAHUo6qKjE-xYECbJQl-PqQzDovcra36anHqPdvLqR9onxVao3rlmvoM6Q3b85sPm7ACiJdjEMbWKUUWypzr6UDzpwpYEcZBAkkZQFtnlZgqcGVRjFolSAEW8qry6lIPURQD_0W00
+
+   Relying Party Solution High Level Architecture
+
+Relying Party Solution Requirements
+=====================================
+
+This section lists the requirements to be met by Relying Parties and Relying Party Solutions.
+
+- The Relying Party MUST register with the Federation Authority to obtain a Registration Certificate that explicitly authorizes which credential types and specific attributes may be requested.
```

Comment:
```suggestion
- The Relying Party MUST register with the Federation Authority to obtain bot the Access Certificate and the Registration Certificate.
```

where we have defined the Access Certificates an d the Registration Certificates we say their intended use and scopes.